### DISCUSSION TITLE
Access Output of Activity by Activity Name Not Working

### DISCUSSION THREAD

**Discussion by manasheed** [elsa-workflows/elsa-guides#25] [Q&A] [answered]
Hi,
I am following [Part 6 of Building Workflow Driven .NET Applications with Elsa 2 tutorial](https://medium.com/codex/part-6-of-building-workflow-driven-net-applications-with-elsa-2-9b3167c612dd) with Elsa 2.3.0. In step 6 I have configured Liquid Engine.
It works fine with immediate following activity with "Input". But When I access variable By Activity name. It does not show any thing in the output. I can share my code and DB if required.
Kindly help me to understand where I am doing something wrong.

**Here is my workflow**
![image](https://user-images.githubusercontent.com/17447195/147630544-09994b4b-b590-4c53-8e35-feb9ecc43ce5.png)

**I have set name of the activity as GetDocument1**
![image](https://user-images.githubusercontent.com/17447195/147630588-8fe99eb4-e078-4a80-b6f4-b9c5cbd236ad.png)

**Here is code to access Output variables using "Input" and by activity name "GetDocument1".** 
![image](https://user-images.githubusercontent.com/17447195/147630601-7b11e30d-51f1-49ec-993c-9ebd4556b180.png)

**Call by activity name does not show file name.**
![image](https://user-images.githubusercontent.com/17447195/147630915-330f5cd9-2dce-4111-b53e-fa972b16ff17.png)

**Comment by sfmskywalker** [comment]
Sure, if you can share your solution + DB then that will definitely help me see what the issue is.

**Reply by manasheed** [reply]
Thanks. Zip file sent via email as it was not being uploaded here.

**Comment by manasheed** [comment, marked as answer]
Upgraded to Elsa version 2.4.1 and issue was resolved.
